gradle/libs.versions.toml

```diff
@@ -1,27 +1,24 @@
 [versions]
 
-gradlePlugin-kotlin = "1.9.22"
+gradlePlugin-kotlin = "2.0.0-Beta4"
 # See: https://kotlinlang.org/docs/gradle-configure-project.html#apply-the-plugin
-gradlePlugin-android = "4.2.2"
-gradlePlugin-android-dokkatoo = "8.0.2"
+gradlePlugin-android = "7.1.3"
 gradlePlugin-dokka = "1.9.20"
-gradlePlugin-gradle-customUserData = "1.12"
-gradlePlugin-gradle-enterprise = "3.14.1"
```

Comment:
I think it makes sense to have the most important / frequently updated dependencies on the top, and it's unlikely that Gradle enterprise will be looked for / updated all that often. Just moved it closer to the end to reduce noise